.github/workflows/release.yaml

```diff
@@ -4,45 +4,35 @@ on:
   push:
     # Sequence of patterns matched against refs/tags
     tags:        
-      - v*
+      - "v*"
 
 jobs:
-  lint:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: ./tools/github-actions/setup-deps
-
-      # lint
-      - run: make lint-deps
-      - run: make -k lint
-
   release:
     runs-on: ubuntu-latest
-    needs: [ lint ]
     steps:
       - uses: actions/checkout@v3
-      - uses: ./tools/github-actions/setup-deps
 
-      - name: Extract Tag value
-        id: tag_env
+      - name: Extract Release Tag and Commit SHA
+        id: vars
         shell: bash
         run: |
-          echo "##[set-output name=version;]$(echo ${GITHUB_REF##*/})"
+          echo "##[set-output name=release_tag;]$(echo ${GITHUB_REF##*/})"
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
 
-      # test
-      - run: make go.test.unit
-
-      # build and push image
       - name: Login to DockerHub
         uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
-
-      - name: Build and Push EG Image
-        env:
-          EG_TAG_VERSION: ${{ steps.tag_env.outputs.version}}
-        run: make image.push.multiarch TAG=${EG_TAG_VERSION} IMAGE_NAME=gateway PLATFORMS="linux_amd64 linux_arm64"
+      # pull, retag gateway-dev image and push
+      - name: Pull gateway-dev image
+        run: |
+          docker pull envoyproxy/gateway-dev:${{ steps.vars.outputs.sha_short }}
```

Comment:
Use `skopeo`, `docker pull` only pulls system arch of the host machine, there's no way to use `docker` to retag a multiarch image.

Reply:
![homer-disappear](https://user-images.githubusercontent.com/9469142/187730490-9f38fbba-2248-4ad4-abe5-a0ac05b957e3.gif)

Reply:
@arkodg if you have no issues with `skopeo`, can you update the PR to resolve @lizan comments?